tests/src/Unit/Plugin/Validation/Constraint/GlobalMessageConstraintValidatorTest.php

```diff
@@ -84,14 +95,34 @@ public function testErrorValidation() {
    * @return \Drupal\Core\Field\FieldItemListInterface|\PHPUnit\Framework\MockObject\MockObject
    *   Mocked field list object.
    */
-  public function getFieldCallback($field_name) {
+  public function getFieldCallback($field_name, bool $validTester = FALSE) {
     $field = $this->createMock(FieldItemListInterface::class);
     if ($field_name == 'su_global_msg_enabled') {
       $field->method('getString')->wilLReturn('foo');
     }
     return $field;
   }
 
+  /**
+   * Mock entity get valid field callback.
+   *
+   * @param string $field_name
+   *   Field machine name.
+   *
+   * @return \Drupal\Core\Field\FieldItemListInterface|\PHPUnit\Framework\MockObject\MockObject
+   *   Mocked field list object.
+   */
+  public function getValidFieldCallback($field_name) {
```

Comment:
this could just be wrapped into the same method as `getFieldCallback` it does the same thing

Reply:
There's one important difference, which is that it'll give you an enabled value, and it'll give you one and only one valid content value. The other function gives you an enabled value and no valid content values.  I just duplicated the callback for convenience.

Reply:
I wrapped your logic into the other callback. the tests do the same thing still, its just less test to maintain. see the [commit diff](https://github.com/SU-SWS/stanford_profile_helper/pull/98/commits/e3e66a8bc7959374c57e2c8ae959df80addd355d) if youre interested